Merchant/Core/Services/SocketService.swift

```diff
@@ -36,10 +36,11 @@ class SocketService {
     
     var addressObs: PublishSubject<WebSocketTransactionResponse>?
     
-    init() {
+    private init() {
         self.ws.event.close = { code, reason, clean in
```

Comment:
weak required.

Reply:
What do you mean? There is a `weak` specified, but wasn't before.